data/blog/electron-11-0.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Electron 11.0.0
+author:
+- VerteDinde
+- sofianguy
+date: '2020-11-17'
+---
+
+Electron 11.0.0 has been released! It includes upgrades to Chromium `87`, V8 `8.7`, and Node.js `12.18.3`. We've added support for Apple silicon, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 11.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). The release is packed with upgrades, fixes, and new support for Apple's upcoming silicon products. 
+
+We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `87.xx`
+    * [New in Chrome 86](https://developers.google.com/web/updates/2020/10/nic86)
+    * [New in Chrome 87]()
+* Node.js `12.18.3`
+    * [Node 12.18.3 blog post](https://nodejs.org/en/blog/release/v12.18.3/)
```

Comment:
We jumped from E10 on node 12.16 to E11 on node 12.18
Let's also add a link to the node 12.17 release blog post: https://nodejs.org/en/blog/release/v12.17.0/